### DISCUSSION TITLE
blog/how-i-learned-to-type-fast/

### DISCUSSION THREAD

**Discussion by giscus** [Jitesh117/blog_comments#4] [Q&A]
# blog/how-i-learned-to-type-fast/

In this article I’m going to write about a topic very close to my heart, which is “How I Learned to type Faster”. Typing fast has been a game changer for me, and I believe anyone who type fasts can become at least twice as productive as they already are.
But why bother to type fast? 1. It just really impresses people When people see me typing while coding, the single biggest complement I’ve got is “Wow!

https://jitesh117.github.io/blog/how-i-learned-to-type-fast/

**Comment by gnaaruag** [comment]
well written blog post!! cheers!!

**Comment by melqtx** [comment]
chad

**Reply by Jitesh117** [reply]
Thanks my G!

**Comment by hyperdevstuff** [comment]
Its all practice after a certain point. dedicated and regular practice is might be the only breakthrough.
